Makefile

```diff
@@ -54,34 +54,42 @@ $(JAVA_CLIENT_JAR): dist/openapi-generator-cli.jar dist/java.swagger.json
 		--artifact-id argo-client-java \
 		--artifact-version $(JAVA_CLIENT_VERSION) \
 		--import-mappings Time=org.joda.time.DateTime \
-		--import-mappings Affinity=io.kubernetes.client.models.V1Affinity \
-		--import-mappings ConfigMapKeySelector=io.kubernetes.client.models.V1ConfigMapKeySelector \
-		--import-mappings Container=io.kubernetes.client.models.V1Container \
-		--import-mappings ContainerPort=io.kubernetes.client.models.V1ContainerPort \
-		--import-mappings EnvFromSource=io.kubernetes.client.models.V1EnvFromSource \
-		--import-mappings EnvVar=io.kubernetes.client.models.V1EnvVar \
-		--import-mappings HostAlias=io.kubernetes.client.models.V1HostAlias \
-		--import-mappings Lifecycle=io.kubernetes.client.models.V1Lifecycle \
-		--import-mappings ListMeta=io.kubernetes.client.models.V1ListMeta \
-		--import-mappings LocalObjectReference=io.kubernetes.client.models.V1LocalObjectReference \
-		--import-mappings ObjectMeta=io.kubernetes.client.models.V1ObjectMeta \
-		--import-mappings ObjectReference=io.kubernetes.client.models.V1ObjectReference \
-		--import-mappings PersistentVolumeClaim=io.kubernetes.client.models.V1PersistentVolumeClaim \
-		--import-mappings PodDisruptionBudgetSpec=io.kubernetes.client.models.V1beta1PodDisruptionBudgetSpec \
-		--import-mappings PodDNSConfig=io.kubernetes.client.models.V1PodDNSConfig \
-		--import-mappings PodSecurityContext=io.kubernetes.client.models.V1PodSecurityContext \
-		--import-mappings Probe=io.kubernetes.client.models.V1Probe \
-		--import-mappings ResourceRequirements=io.kubernetes.client.models.V1ResourceRequirements \
-		--import-mappings SecretKeySelector=io.kubernetes.client.models.V1SecretKeySelector \
-		--import-mappings SecurityContext=io.kubernetes.client.models.V1SecurityContext \
-		--import-mappings Toleration=io.kubernetes.client.models.V1Toleration \
-		--import-mappings Volume=io.kubernetes.client.models.V1Volume \
-		--import-mappings VolumeDevice=io.kubernetes.client.models.V1VolumeDevice \
-		--import-mappings VolumeMount=io.kubernetes.client.models.V1VolumeMount \
+		--import-mappings Affinity=io.kubernetes.client.openapi.models.V1Affinity \
```

Comment:
Can you drop in a quick comment explaining why openapi instead of not? Just to help anyone in the future who might see some reason to switch back.

Reply:
you mean add a comment to say the packages in imports need to match the dependency in the pom?  Just trying to understand who is the target audience for the comment.

Reply:
Ah! Gotcha. Thought these were two namespaces in the same depencency. No comment is fine then.